### DISCUSSION TITLE
Module not found: Can't resolve '@xxx' when I run docker image

### DISCUSSION THREAD

**Discussion by thomas-void0** [vercel/next.js#57507] [Help]
### Summary

**report a error: Module not found: Can't resolve '@xxx'  when I build docker image.**
such as:
<img width="1064" alt="image" src="https://github.com/vercel/next.js/assets/48620706/d9fb8a0c-7c16-4dc7-bbf8-3ce9b9346240">

**this is my dockerfile:**
<img width="847" alt="image" src="https://github.com/vercel/next.js/assets/48620706/4f315870-2f6e-4ff2-af7b-9ff3a83ec0fe">

**Strangely enough, it worked when I executed the following dockerfile:**

<img width="891" alt="image" src="https://github.com/vercel/next.js/assets/48620706/05ca89d5-e986-4e35-b3c5-90abd827ed87">

**The only difference between them is that one is a test and one is formal. this is my pkg.json:**

<img width="642" alt="image" src="https://github.com/vercel/next.js/assets/48620706/ff117aba-9008-47de-adba-23b8b204f569">

I try change version of next slove this question. but it still exists. my project is run `npx create-next-app@latest` generate.

Looking forward to your reply.

**Comment by juanvilladev** [comment]
Have you found an answer to this? I still can't deploy my latest changes because of this.

**Reply by nam-diijam** [reply]
any update?

**Comment by rizshivalli** [comment]
For me it as the capital case issue.

in local it was `./src/components ` but in BitBucket it was `./src/Components`.

once i renamed it, it was resolved

I followed the following method

```
# Rename the folder to a temporary name
git mv ./src/Components ./src/tempname

# Rename the folder back to 'components' with the desired case
git mv ./src/tempname ./src/components

```

**Comment by YingjieHe** [comment]
For me, it seems like I need to install typescript before setting NODE_ENV=production in the dockerfile

**Reply by n0bitaemon** [reply]
Can you clarify steps you did? i'm having the same problem
